react-todo-app/src/components/TodosList.js

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import TodoItem from "./TodoItem";
+
+const TodosList = props => {
```

Comment:
[Optional] I think it would be better if you destruct props to keep your code clean.

``` const TodosList = ({id, handleChangeProps, deleteTodoProps, setUpdate}) => {} ```